plugins/kn-admin/README.md

```diff
@@ -0,0 +1,66 @@
+# kn-admin
+A plugin for kn client for management
+This kn-admin plugin is designed to help administrators or operators better manage a Knative platform installation with kn CLI. 
+The plugin’s main objective is to make administration and operation workflows easier, for instance by making it easy to accomplish 
+tasks such as feature flags enablement or disablement with one command,instead of many manual steps like modifying ConfigMaps or yaml files.
+
+## Build
+To build kn-admin
+```bash
+$ pwd
+/Users/zhanggong/zhgworkspace/knative/src/github.com/knative/client-contrib/plugins/kn-admin
+$ ./hack/build.sh
+🧹 Format
+🚧 Compile
+$ ls build/_output
+  kn-admin
+
+```
+To run kn-admin as plugin, please refer to https://github.com/knative/client/blob/master/docs/README.md#options
+
+## kn-admin Usage
+```bash
+$ kn admin -h
+A plugin of kn client to manage Knative for administrators. 
+
+For example:
+kn admin domain set - to set Knative route domain
+kn admin private-registry enable - to enable deployment from the private registry
+
+Usage:
+  admin [command]
+
+Available Commands:
+  domain           Manage route domain
+  help             Help about any command
+  private-registry Manage private-registry
```

Comment:
Might be good to add `version` command

Reply:
I'll add it in next PR and follow kn client version cmd.